lib/torque/animator.js

```diff
@@ -1,4 +1,5 @@
 var torque = require('./');
+var StepsRange = require('./animator-steps-range');
```

Comment:
it's hard to follow when the class is named "AnimatorStepsRange" internally and "StepsRange" here